src/sql/luonnit.sql

```diff
@@ -28,9 +28,14 @@ CREATE TABLE Paivitykset (
     Segmentti BIGINT UNSIGNED,
     Aika DATETIME,
     Lumilaatu INT,
-    Teksti TEXT,
+    Kuvaus TEXT,
+    Lumen_kuva BLOB,
+    Lumilaatu_ID bigint(20) unsigned,
```

Comment:
There are Lumilaatu and Lumilaatu_ID in table Paivitykset, and both of these are referring to snowtype ID.